pkgs/tools/misc/octofetch/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "octofetch";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "azur1s";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-ciBFTVdHotjifNAoXJjI9CumyK98OkMmfWMbrEldlNI=";
+  };
+
+  cargoSha256 = "sha256-Gzemm5HY6YwlxesQlil6R+34OtAeU2k7f/9+Lll3i8k=";
+
+  nativeBuildInputs = [ pkg-config openssl ];
+
```

```suggestion
  nativeBuildInputs = [ pkg-config openssl ];
```

Is openssl only linked against or do we need some binary? If we need no binary please move it to buildInputs. Also the envs in preBuild are maybe already enough and we don't need an entry here at all.